packages/fuels-accounts/src/provider.rs

```diff
@@ -707,7 +711,7 @@ impl DryRunner for Provider {
     async fn dry_run(&self, tx: FuelTransaction) -> Result<DryRun> {
         let [tx_execution_status] = self
             .client
-            .dry_run_opt(&vec![tx], Some(false))
+            .dry_run_opt(&vec![tx], Some(false), Some(0))
```

Comment:
We should either use `dry_run_no_validation` or remove it in favor of dry_run_opt